reconcile/openshift_rolebindings.py

```diff
@@ -90,6 +92,14 @@ def fetch_desired_state(ri, oc_map):
     roles = gqlapi.query(ROLES_QUERY)['roles']
     users_desired_state = []
     for role in roles:
+        expDate = openshift_resource.checkExpirationDate(role)
+        if expDate is True:
+            continue
+        elif expDate is False:
+            raise ValueError(
+                f'The maximum expiration date of {role["name"]} '
+                f'shall not exceed {openshift_resource.EXPIRATION_MAX} \
+                    days from today')
```

Comment:
by failing here for a wrongly configured expiration date, other good rolebindings are not reconciled for the time being. i know we do this in other places as well but if you could find a way to not generally fail the reconcile run but do whatever work can be done that would be great. 

BUT i haven't looked at how easy that can be done and i don't want to block a great enhancement to qontract-reconcile. so if you decide not to implement this right now, it is fine with me but leave a comment in the code and create a follow up jira ticket.

Reply:
Same comment as above. Plus, what @geoberle mentioned.

Reply:
I will go ahead and follow that up with a jira ticket.